composer/trainer/trainer.py

```diff
@@ -117,7 +121,8 @@ def __init__(
             *,
             model: BaseMosaicModel,
             train_dataloader: Union[DataLoader, DataSpec],
-            eval_dataloader: Union[DataLoader, DataSpec],
+            eval_dataloader: Optional[Union[DataLoader, DataSpec]] = None,
```

Comment:
```suggestion
```
I'd be for specifying just the evaluators, not the eval dataloader

Reply:
Can you also remove the hparams for valiadate_every_n_epochs and validate_every_n_batches?